src/kodeagent/kodeagent.py

```
@@ -890,7 +890,7 @@ async def _act(self) -> AsyncIterator[AgentResponse]:
                     f'Args provided: {tool_args_dict}\n'
                     f'Please check the tool signature and try again with correct arguments.'
                 )
-                logger.exception(error_msg)
+                logger.error(error_msg)
```

Ruff (0.14.6)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)